sv_pipeline/exome/load_data.py

```diff
@@ -4,21 +4,19 @@
 import logging
 import os
 import re
-import subprocess
 
-from datetime import datetime
 from elasticsearch import helpers as es_helpers
 from getpass import getpass
 from tqdm import tqdm
 
 from hail_scripts.shared.elasticsearch_client_v7 import ElasticsearchClient
 from hail_scripts.shared.elasticsearch_utils import ELASTICSEARCH_INDEX
 
+from sv_pipeline.utils.common import get_sample_subset, get_sample_remap, get_es_index_name, CHROM_TO_XPOS_OFFSET
+
```

Comment:
Some common utils functions have been moved to a common file. The exome SVs loading pipeline itself has no changes.